vnext/Shared/Utils.h

```diff
@@ -22,4 +22,7 @@ struct Url {
 
 winrt::Windows::Foundation::IAsyncOperation<winrt::hstring> getApplicationDataPath(const wchar_t *childfolder);
 
+// string formatting
+std::string string_format(_Printf_format_string_ const char *format, ...);
```

Comment:
The `string_format` name does not play well with other naming conventions that we have. We mostly use the Pascal naming conventions. Also, the function should typically start with a verb. Could you please change it to `FormatString`?